apps/site/mix.exs

```diff
@@ -112,6 +113,8 @@ defmodule Site.Mixfile do
       {:poolboy, "~> 1.5"},
       {:wallaby, "~> 0.22", runtime: false, only: :test},
       {:con_cache, "~> 0.12.0"},
+      {:recaptcha,
+       github: "samueljseay/recaptcha", ref: "8ea13f63990ca18725ac006d30e55d42c3a58457"},
```

Comment:
@digitalcora I consider it undesirable as a rule to point to some random SHA in a dep, but I'm guessing we need something not yet included in a release? If that's so could you please leave a `FIXME` comment here, explaining what that not-yet-included stuff is, and noting that we should switch to a proper release of `recaptcha` when that's possible?

Reply:
I consider it undesirable as well, but this is unfortunately the only way we can make the controller logic testable, since the ability to artificially generate an error response from the `MockClient` doesn't exist in any released version of the package. I can add a comment explaining this.